src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1074,11 +1076,13 @@ def run_dm_activities_with_ior(self, tool, pool, cont, create_dataset=False):
                 dst_path=format_path(pool, cont3))
             read_back_cont = cont3
             read_back_pool = pool
-        # the result is that a NEW directory is created in the destination
-        if tool == 'FS_COPY':
-            daos_path = "/" + os.path.basename(posix_path) + self.ior_cmd.test_file.value
+        if tool in ['FS_COPY', 'DCP']:
+            # the result is that a NEW directory is created in the destination
+            daos_path = os.path.join(os.sep, os.path.basename(posix_path), 'testfile')
+        elif tool in ['CONT_CLONE', 'DSERIAL']:
+            daos_path = os.path.join(os.sep, 'testfile')
```

Comment:
We should still use `self.ior_cmd.test_file.value` to build the path instead of hardcoding `testfile`. If the leading slash is a problem, `self.ior_cmd.test_file.value.lstrip(os.sep)` should help

Reply:
The problem is not leading slash.
The issue is that (from what I remember) the attribute `self.ior_cmd.test_file.value` is modified during the test and lead to some errors at the end of the test.

Reply:
From what I remember, the issue here is that the file to read with tool in ['CONT_CLONE', 'DSERIAL'] is not equal to the value of the property `self.ior_cmd.test_file`: this property is equal to `daos:testFile`. The container which contains the file to read has been previously used to write data with ior with a hardcoded filename equal to 'testfile'.

Reply:
Ah, I see what you mean. I'm okay with that. It would be too much undue work to fix in this PR